interfaces/apparmor/backend.go

```diff
@@ -391,32 +389,44 @@ func (b *Backend) Setup(snapInfo *snap.Info, opts interfaces.ConfinementOptions,
 	// the cache (since we know those changed for sure).  This allows us to
 	// work despite time being wrong (e.g. in the past). For more details see
 	// https://forum.snapcraft.io/t/apparmor-profile-caching/1268/18
-	pathnames := make([]string, len(changed))
+	changedPaths = make([]string, len(changed))
 	for i, profile := range changed {
-		pathnames[i] = filepath.Join(dir, profile)
+		changedPaths[i] = filepath.Join(dir, profile)
 	}
 
-	var errReloadChanged error
-	timings.Run(tm, "load-profiles[changed]", fmt.Sprintf("load changed security profiles of snap %q", snapInfo.InstanceName()), func(nesttm timings.Measurer) {
-		errReloadChanged = loadProfiles(pathnames, cache, skipReadCache)
-	})
-
 	// Load all unchanged profiles anyway. This ensures those are correct in
```

Comment:
is this comment still accurate? AFAICT we aren't loading any profiles in this function anymore